scripts/release.sh

```diff
@@ -90,7 +110,7 @@ function tag_images() {
     # Creating a local tag so that images are uploaded with it
     git tag -a -f "${release['version']}" -m "${release['version']}"
 
-    make release RELEASE_ARGS="$images --tag ${release['version']}"
+    dryrun make release RELEASE_ARGS="$images --tag ${release['version']}"
```

Comment:
Given that `release.sh` handles `--dryrun` itself, is it necessary to protect `make release`?

Reply:
I'm not sure what do you mean here?

Reply:
I don’t understand why we don’t want to actually run `make release` here in dry-run mode (passing the appropriate `using` settings). If the point of dry-run mode is to check as much as possible without making any externally-visible changes, I wonder if it wouldn’t be better to run `make release` here, since `release.sh` itself is now capable of not making externally-visible changes.

Reply:
`make release` will attempt to push images to quay, why would we run it?

Reply:
If the goal is “helping testing of the operations of the release process” as described in the commit message, it seems to me we should be testing as much as possible of the release process. Given that `release.sh` itself _won’t_ push to Quay in dry-run mode, that protection should be sufficient, so I think we should run `make release` and let `release.sh` deal with not doing stuff we don’t want. You know that `make release` itself isn’t useful in dry-run mode, but I’d still like that to be tested.

Reply:
Put another way, I think dry-run mode should skip the fewest commands possible.

Reply:
`make release` pushes to quay.
You can try run it locally and see that it errors the `make create-release` call.

As I see it, if you run in "dry run" mode you shouldn't be getting errors, unless something is wrong with your yamls.

Furthermore, `make release` is part of shipyard and hence considered external to this process, indeed it doent make sense to test it in the dry run mode.
We could, if it makes more sense to you, simply call `docker push` here and prefix it with `dryrun` but I see no point in duplicating the code..

Reply:
> Furthermore, `make release` is part of shipyard and hence considered external to this process, indeed it doent make sense to test it in the dry run mode.

Ah sorry, I’d missed that, I thought it would end up running `release.sh` here.